resources/icons/svg/svgicons.qrc

```diff
@@ -1,169 +1,169 @@
 <RCC>
```

Comment:
Please remove this file from this PR.
It has nothing do to with the changes

Reply:
The file reverted to the previous version.